.golangci.yml

```diff
@@ -49,6 +49,7 @@ linters:
     - structcheck
     # - stylecheck  # instead of golint
     # - testpackage
+    - thelper
```

Comment:
This is related to the Go testing framework however we're migrating all UTs to Ginkgo. Eventually all the functions to which you added `t.Helper() `will go away. Therefore I don't think we need that PR.

Reply:
Okay, closing. Thanks Tom!